### ISSUE TITLE
User Cannot See Items in the Inventory Manager Tab

### ISSUE THREAD

**Issue by loesinga** [jrattanpal/InventoryManagerNP#8]
He can see the Home Page, Items, Item Actions, Locations just fine - but when he goes to the Inventory Manager tab, the page is blank. He can see the "Create New Item" and "Refresh Data" buttons, but there is nothing underneath that. I have attempted:

Adding him to the Inventory Manager Permission set (he was already added)
Giving him access to all Apex Classes associated with the installed package
Adding his profile to selected profiles for the Inventory Manager app in App Manager
Making him the owner of all items that he needs to see.

Still, the Inventory Manager page is blank for him. I, however, can see all the items in the Inventory Manager because I am a System Admin. Any ideas on what to troubleshoot here?

**Comment by jrattanpal** [comment]
Can you capture some Apex debug logs for that and share with me?

Although, you may have to log a case to enable namespace debugging for "invmgrnp" namespace. 

You seem to have gone through all the steps that are needed so I am not sure what may be missing

**Comment by loesinga** [comment]
The Permission Set provided with the App did not seem to have enough permissions for full functionality. First, errors were arising on the Inventory Manager tab related to Apex Class access. To resolve this, I had to clone the permission set and add the related Apex Class access to it.

Then, the list of items with +/- were not appearing on the Inventory Manager page. What ended up resolving this was again related to the permission set. I added access to the permission set for the Location and SubType objects, and this resolved the issue. Although, I made these changes to object access at the same time, so I am not sure if one of the Locations/Subtype objects or both were causing the issue.
